src/components/utils/useReducedMotion.spec.ts

```diff
@@ -1,6 +1,7 @@
 import useReducedMotion from './useReducedMotion';
-import {act, renderHook} from '@testing-library/react-hooks';
+import {renderHook} from '@testing-library/react';
 import MatchMedia from './__mocks__/mq';
+import {act} from 'react';
```

Comment:
I think, we should do similar refactoring as it was done here https://github.com/brainly/frontend-monorepo/pull/516
I mean, import `act` from `'@testing-library/react';` and remove `@testing-library/react-hooks` from the codebase